students/Tim/mailroom/mailroom.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Nov  6 08:45:27 2019
+
+@author: tdietz
+"""
+import sys
+donorList = [("Elon Musk", [45643.05, 60029090.04]), 
+             ("Hank Hill",[23.00, 457.47]), 
+             ("Wade Watts", [430594.56, 420.69]),
+             ("Stan Lee", [345698.45, 3490938.56]),
+             ("Col Mustard", [4345.45, 3434.46]),
+             ("Mark Normand", [45.00]),
+            ]
+
+def createReport():
+    print("Create Report Option")
+    #TODO create a function that generates a report
```

Comment:
You still do not have any functionality here. Do you need any help how to start?